CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 To install Beacon, see the [documentation](https://developer.helpscout.com/beacon-2/mobile/).
 
+## 3.0.0 Release (TBD) 
+
+#### Bug fixes
```

Comment:
This isn't really a bug fix, but rather adding support for a new OS. For consistency, we should put it under **Features**.
```suggestion
#### Features
```